apps/client/src/styles/index.css

```diff
@@ -0,0 +1,6 @@
+#root {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
```

Comment:
Necessary? I believe Cloudscape app frame handles this.

Reply:
Not handled by the app frame unfortunately.